### DISCUSSION TITLE
No frame laser?

### DISCUSSION THREAD

**Discussion by Peek-a-Paulien** [arkypita/LaserGRBL#1581] [Q&A]
*Sorry for broken English, because Dutch*
I want to use the button Frame to see if my material is rightly alligned. When I click on Frame, it does trace the workspace, but not with the laserlight on. It has to be perfectly alligned, so I don't trust this at the moment... 
Any ideas how I can switch the laser during Frame on? 

Thanks!

**Comment by Tribo-2dot0** [comment]
Right click and choose Edit Button. Put S with a minimum value for your laser to light up (in mine the value has to be at least 4), maybe that's it.

**Comment by amadensor** [comment]
I turn on focus mode before I hit frame.   It is bright enough to see, but doesn't etch anything.
